packages/gitlab/dev/mock-gitlab/api-v4-v15.7.0.ts

```diff
@@ -1,5 +1,55 @@
+const graphqlProjectCoverageQuery = `
+    query getProjectCoverage(
+        $projectSlug: ID!
+        $updatedAfter: Time
+    ) {
+        project(fullPath: $projectSlug) {
+            name
+            webUrl
+            pipelines(ref: "main", updatedAfter: $updatedAfter) {
```

Comment:
Here, we cannot know if the default branch is `main`, we have to get the `default_branch` from API.

Reply:
Updated with separate call